src/components/common/TmBalance.vue

```diff
@@ -368,8 +370,17 @@ export default {
       this.showTutorial = false
     },
     setPreferredCurrency() {
-      localStorage.setItem(`preferredCurrency`, this.selectedFiatCurrency)
-      this.preferredCurrency = this.selectedFiatCurrency
+      this.$store.dispatch(`setPreferredCurrency`, this.preferredCurrency)
+    },
+    getPreferredCurrency() {
+      const localPreferences = localStorage.getItem(USER_PREFERENCES_KEY)
```

Comment:
move into store?

Reply:
actually it was originally there and then I thought it was too specific 😅  I had an inner battle there. Let's move it back